packages/replay/src/replay.ts

```diff
@@ -198,6 +198,23 @@ export class ReplayContainer implements ReplayContainerInterface {
         // instead, we'll always keep the last 60 seconds of replay before an error happened
         ...(this.recordingMode === 'error' && { checkoutEveryNms: ERROR_CHECKOUT_TIME }),
         emit: this._handleRecordingEmit,
+        onMutation: (mutations: unknown[]) => {
```

Comment:
I think you should be able to `import {mutationRecord} from '@sentry-internal/rrweb'`

Reply:
IMHO we can leave this as unknown as we really don't care about what type of array that is (for now), one less import to care about 😅